task_5/Toma.Lobanova/src/main/java/ua/od/hillel/SequenceTask2/Sequence.java

```diff
@@ -0,0 +1,10 @@
+package ua.od.hillel.SequenceTask2;
+
+import java.util.stream.Stream;
+import java.util.stream.Collectors;
+
+public class Sequence {
+    public static void main(String... args){
+        Stream.iterate(1, item -> item + 2).limit(55).peek(item -> System.out.println(item)).collect(Collectors.toList());
```

Comment:
We will look at collectors as well a bit later. But feel free to use everything you find suitable, I will correct if something goes wrong :)